.flake8

```diff
@@ -1,25 +1,53 @@
 [flake8]
-ignore=
-  # Black may add spaces in slice[func(v) : end] syntax
-  E203,
-  # Black leaves commas after combining lines
-  E231,
-  # Allow import * (for settings)
-  F403,
-  F405,
-  # Allow lambdas:
-  E731,
-  # line break before binary operator (incompatible with black):
-  W503,
-  # Allow raise Exception()
-  R102,
-exclude=.git,
-        **/migrations/*,
-        .env,
-        docs,
-        scripts,
-format = ${cyan}%(path)s${reset}:${yellow_bold}%(row)d${reset}:${green_bold}%(col)d${reset}: ${red_bold}%(code)s${reset} %(text)s
-max-complexity=8
+# A = builtins
```

Comment:
Kan dit naar aparte PR getrekt worden samen met alle andere precommit tweaks?

Reply:
Yes, I could do that. But why if I may ask so? I tightened the checks so that the new code I would write as part of this PR would be fine.

Reply:
For clarity. 
Just like with commit messages, separation of logic introduced into repository should be logical.
Code checks have absolutely nothing to do with notifications, but as they are introduced in same PR - some may get confused and think that they are related and notifications code may need tighter checks.

Obviously ^ will never happen, but it's not a reason to put unrelated changes into single PR.

Reply:
Ah, I get it now. Thank for the explanation.

Personally I am okay with PR's that include changes that are tangentially related to the topic of the PR (in this case I wanted my new code to adhere to the strictest style checks). As long as the changes are introduced in separate commits (with clear commit messages :-)). After all the commits are what is normally retained after the PR has been_rebased and merged_. After all the commits are part of the Git history, the PR is not (its part of the Git{Hub,Lab} history. So after a _rebase and merge_ you can't see whether the changes were introduced by one or two PRs.

However for reviewing purpose it might make sense to split them up. Hence I'll do so.

We should definitely discuss this more. Small PRs and _squash and merge_. Larger PRs and _rebase and merge_.

Reply:
Done, moved the tighter `flake8` and `pre-commit` check to a new PR: https://github.com/Amsterdam/dataservices-airflow/pull/272